Formula/herrie.rb

```diff
@@ -1,22 +1,20 @@
 class Herrie < Formula
   desc "Minimalistic audio player built upon Ncurses"
-  homepage "https://herrie.info/"
+  homepage "https://github.com/EdSchouten/herrie"
   url "https://github.com/EdSchouten/herrie/releases/download/herrie-2.2/herrie-2.2.tar.bz2"
   sha256 "142341072920f86b6eb570b8f13bf5fd87c06cf801543dc7d1a819e39eb9fb2b"
   license "BSD-2-Clause"
   revision 1
 
-  livecheck do
-    url "https://github.com/EdSchouten/herrie"
-  end
-
   bottle do
     sha256 catalina:    "ceb3bb01f7cc36c3d55d785ff6d06706708ce11e81bd9da29b4dcdaaa6476f9a"
     sha256 mojave:      "689038931cdc86fee22b0e4217bdb4eccc5c7591254c67300aade0637ae1dc5e"
     sha256 high_sierra: "65631ae69a925bf48a3e4167807b13c36988db1bd1d4f7ed311356da39344c9a"
     sha256 sierra:      "f794117e72309d83bc947463032c262a7341d21eabc5099712016d0f078804f2"
   end
 
+  deprecate! date: "2021-02-09", because: :repo_archived
```

Comment:
Is this when it was archived?

Reply:
I have no idea.  Is there a way to tell?

It is the date I *noticed* it was archived, if that means anything :-)

Reply:
Is there a way to find out the date when a repo is archived? The [docs](https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae#deprecation) say to use today's date if one isn't clear, which is almost certainly what happened here.

Reply:
> Is there a way to find out the date when a repo is archived? The [docs](https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae#deprecation) say to use today's date if one isn't clear, which is almost certainly what happened here.

Again, if there is a way to get this information from github I don't know it.

I scrolled back a few years in @EdSchouten's activity timeline but I don't see it marked there.

I think today's date is fine.  The project had a release in 2018 so it's hardly palaeolithic.  Letting its deprecation clock start now seems fine.

Reply:
>if there is a way to get this information from github I don't know it.

Yes, you've made that very clear. I was asking @SMillerDev.

Reply:
I thought you could in the UI, guess not. https://api.github.com/repos/EdSchouten/herrie has `last_pushed` data, but I guess the current date is fine.